Core/Extensions/ConfigurationExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace PlexBot.Core.Extensions;
+
+public static class ConfigurationExtensions
+{
+    public static string GetRequiredValue<T>(this IConfiguration configuration, string configurationValueName)
+    {
+        var value = configuration[configurationValueName];
```

Comment:
Don't use var only explicit types. There are also some in DiscordHelpers and Program.

Reply:
I have no idea what .editorconfig does. Im showing my lack of knowledge again lol 

I will look it up and learn about it before we make one.